Setting up a fresh hydra installation results in panic

**Describe the bug**
Trying to test something in hydra _(does the script test-e2e-jwt.sh work when NOT using the memory databse? May need to open separate issue)_ and get an error when trying to setup the tables:

```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/ory/hydra/cmd/cli.getDBUrl(0x160a560, 0x167bf28, 0x0, 0x0, 0x0, 0xc4200f4d00, 0xc420223c78)
	/home/workspace/src/github.com/ory/hydra/cmd/cli/handler_migrate.go:100 +0x39a
github.com/ory/hydra/cmd/cli.(*MigrateHandler).MigrateSQL(0xc42000e720, 0x160a560, 0x167bf28, 0x0, 0x0)
	/home/workspace/src/github.com/ory/hydra/cmd/cli/handler_migrate.go:149 +0x62
github.com/ory/hydra/cmd/cli.(*MigrateHandler).MigrateSQL-fm(0x160a560, 0x167bf28, 0x0, 0x0)
	/home/workspace/src/github.com/ory/hydra/cmd/migrate_sql.go:43 +0x52
...
```

**To Reproduce**
Steps to reproduce the behavior:
1. mkdir -p $HOME/workspace/hydra
2. export GOPATH=$HOME/workspace/hydra
3. go get -u -d github.com/ory/hydra
4. cd $GOPATH/src/github.com/ory/hydra
5. export GO111MODULE=on
6. go install .
7. docker run --name postgres -e POSTGRES_USER=hydra -e POSTGRES_PASSWORD=secret -e POSTGRES_DB=hydra -d -p 5432:5432 postgres:9.6
8. DATABASEURL="postgres://hydra:secret@localhost:5432/hydra?sslmode=disable" $GOPATH/bin/hydra migrate sql

**Expected behavior**
No panics - tables created

**Version:**
 - Environment: [e.g. Docker, Kubernetes, ...]: Ubuntu 18.04.1/ go version go1.11.1 linux/amd64
 - Version [e.g. v1.0.0]: Commit 48b2a2278cc371d800891dd4a3ffeea9322a6140

--- a/cmd/cli/handler_migrate.go
+++ b/cmd/cli/handler_migrate.go
@@ -92,7 +92,7 @@ func getDBUrl(cmd *cobra.Command, args []string, position int) (dburl string) {
 		}
 		dburl = viper.GetString("DATABASE_URL")
 	} else {
-		if len(args) < position {
+		if len(args) <= position {
 			fmt.Println(cmd.UsageString())
 			return
 		}

--- a/cmd/root_test.go
+++ b/cmd/root_test.go
@@ -123,7 +123,7 @@ func TestExecute(t *testing.T) {
 		{args: []string{"keys", "import", "--endpoint", backend, "import-2", "../test/stub/rsa.key", "../test/stub/rsa.pub"}},
 		{args: []string{"token", "revoke", "--endpoint", frontend, "--client-secret", "foobar", "--client-id", "foobarbaz", "foo"}},
 		{args: []string{"token", "client", "--endpoint", frontend, "--client-secret", "foobar", "--client-id", "foobarbaz"}},
-		{args: []string{"help", "migrate", "sql"}},
+		{args: []string{"migrate", "sql"}},
 		{args: []string{"version"}},
 		{args: []string{"token", "flush", "--endpoint", backend}},
 	} {

